### ISSUE TITLE
Unable to send tabs

### ISSUE THREAD

**Issue by danielmapar** [docusign/docusign-node-client#50]
This issue is related to the post: https://github.com/docusign/docusign-csharp-client/issues/75

I know this is a C# post, but I am going through the exact same issue using the NPM package.

It gives me a "Bad Request" error... I did the same steps on my admin account, got the 'userinfo' CustomField and the code for Tabs.

![screen shot 2016-05-19 at 6 55 13 pm](https://cloud.githubusercontent.com/assets/4336805/15410988/47b61850-1df3-11e6-970e-9221a1fa2149.png)

Here is the code:

```javascript
// create a template role with a valid templateId and roleName and assign signer info
var tRole = new docusign.TemplateRole();
tRole.setRoleName(docusignFields.templateRoleName);
tRole.setName(docusignFields.fullName);
tRole.setEmail(docusignFields.recipientEmail);
tRole.setTabs(new docusign.Tabs());
tRole.getTabs().setTextTabs([]);

var textTab = new docusign.Text();
textTab.setTabLabel("userinfo");
textTab.setValue("IT WORKS!!!");
tRole.getTabs().getTextTabs().push(textTab);

// create a list of template roles and add our newly created role
var templateRolesList = [];
templateRolesList.push(tRole);

// assign template role(s) to the envelope
envDef.setTemplateRoles(templateRolesList);

// send the envelope by setting |status| to 'sent'. To save as a draft set to 'created'
envDef.setStatus('sent');

// use the |accountId| we retrieved through the Login API to create the Envelope
var loginAccount = new docusign.LoginAccount();
loginAccount = loginAccounts[0];
var accountId = loginAccount.accountId;

// instantiate a new EnvelopesApi object
var envelopesApi = new docusign.EnvelopesApi();

// call the createEnvelope() API
envelopesApi.createEnvelope(accountId, envDef, null, function (err, envelopeSummary, response) {

    if (err) {
        defer.reject("[DOCUSIGN] Error: " + err);
    } else {
        console.log('EnvelopeSummary: ' + JSON.stringify(envelopeSummary));
        defer.resolve(envelopeSummary);
    }

});```

Any ideas?

**Comment by danielmapar** [comment]
Never mind, fixed the issue by following: http://stackoverflow.com/questions/36871943/docusign-request-returns-error-only-when-tabs-set

**Comment by mmallis87** [comment]
It's a known issue that should be fixed, once we release a new version of the Node package. The DocuSign Signature REST API doesn't like null objects :)

I'm closing the issue since you get the fix and it's a dupe of #47
